pkg/cli/cmd/run/run.go

```diff
@@ -215,3 +267,20 @@ func (r *Runner) displayPortforwardMessages(status <-chan portforward.StatusMess
 		fmt.Printf("%s %s [port-forward] %s from localhost:%d -> ::%d\n", regular.Sprint(message.ReplicaName), bold.Sprint(message.ContainerName), message.Kind, message.LocalPort, message.RemotePort)
 	}
 }
+
+// dashboardDeploymentExists checks if a dashboard deployment exists in the given Kubernetes context.
+func dashboardDeploymentExists(ctx context.Context, kubernetesClient k8sclient.Interface, kubeContext string, dashboardLabelSelector labels.Selector) bool {
```

Comment:
We do not need to pass kubeContext now.
```suggestion
func dashboardDeploymentExists(ctx context.Context, kubernetesClient k8sclient.Interface, dashboardLabelSelector labels.Selector) bool {
```